### PR TITLE
Remove `redux-form`

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

Finally!

# Result

No visual changes

# Testing

- [x] I have thoroughly tested my changes.

Tested all checkboxes and radio buttons, and all refactored forms. The join event form was pretty thoroughly tested.

### CODE REVIEW

File: app/components/Form/Field.tsx
Comment (norbye) [inline app/components/Form/Field.tsx:1]: Is the warning functionality you have removed here implemented somewhere else, or are warnings simply not supported any more?

File: app/components/LoginForm/ForgotPasswordForm.tsx [hunk 1]
```diff
@@ -1,86 +1,56 @@
...
+});
 
-class ForgotPasswordForm extends Component<Props, State> {
-  mounted = false;
-  state = {
-    submitted: false,
-  };
+const ForgotPasswordForm = () => {
+  const [mounted, setMounted] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
```
Comment (norbye) [inline app/components/LoginForm/ForgotPasswordForm.tsx:14]: Might be possible to get these values from `react-final-form` through the formState https://final-form.org/docs/final-form/types/FormState

File: app/components/LoginForm/RegisterForm.tsx [hunk 2]
```diff
@@ -1,96 +1,72 @@
...
+});
 
-class RegisterForm extends Component<Props, State> {
-  mounted = false;
-  state = {
-    submitted: false,
-  };
+const RegisterForm = () => {
+  const [mounted, setMounted] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
```
Comment (norbye) [inline app/components/LoginForm/RegisterForm.tsx:22]: Same here

File: app/components/LoginForm/RegisterForm.tsx [hunk 3]
```diff
@@ -1,96 +1,72 @@
...
+        <Form onSubmit={handleSubmit} onClick={(e) => e.stopPropagation()}>
+          <Field
+            name="email"
+            component={TextInput.Field}
+            placeholder="E-post"
+          />
+          <Field
+            name="captchaResponse"
+            fieldStyle={{
+              width: 304,
```
Comment (norbye) [inline app/components/LoginForm/RegisterForm.tsx:59]: Doesn't feel too dynamic(?)

**Review by norbye — APPROVED** [review]
Awesome stuff, good job!!

My comments are just small stuff - so do with it as you want

File: app/components/LoginForm/RegisterForm.tsx (hunk 3 above)
Comment (ivarnakken) [reply app/components/LoginForm/RegisterForm.tsx:59]: Yup but I didn't want to bother improving css here

File: app/components/LoginForm/ForgotPasswordForm.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/components/LoginForm/ForgotPasswordForm.tsx:14]: Yes probably - I'll look into it tomorrow

Comment (ivarnakken) [reply app/components/LoginForm/ForgotPasswordForm.tsx:14]: Looked into it now. It is possible to look at the submitted state, but it's not possible to get and use these outside of the form.

**Pull request merged**